index.py

```diff
@@ -187,11 +209,11 @@ def on_message(client, userdata, message):
     # try to get plate number
     plate_number = None
     score = None
+
     if config.get('plate_recognizer'):
         plate_number, score = plate_recognizer(response.content)
     else:
-        _LOGGER.error("Plate Recognizer is not configured")
```

Comment:
I'd put an elif here to check for your config then an else with the original error and return